game_map.py

```diff
@@ -79,8 +82,10 @@ def render(self, console: Console) -> None:
         Otherwise, the default is "SHROUD".
         """
         console.tiles_rgb[0 : self.width, 0 : self.height] = np.select(
-            condlist=[self.visible, self.explored],
-            choicelist=[self.tiles["light"], self.tiles["dark"]],
+            #condlist=[self.visible, self.explored],
+            condlist=[self.aoe, self.visible, self.explored],
+            #choicelist=[self.tiles["light"], self.tiles["dark"]],
+            choicelist=[self.tiles["aoe"], self.tiles["light"], self.tiles["dark"]],
```

Comment:
I don't think a new graphic needs to be added to the tile data for this effect.

The `aoe` should conditionally apply an effect instead of being added to `np.select`.

Also try not to commit commented out code in the future.

Reply:
Sorry about the commented code.  Missed them when I cleaned up the edits.

I was actually going to add it in as an effect.   I changed to adding adding it as a graphic for the tile data since it would allow for using a different graphic for each tile type.  I used it have the walls show in a different color if you chose to include walls in the aoe calculation.   If for example you wanted additional floor types, you could render the aoe overlay differently for each one if you wanted to.

I also chose to implement the overlay in the engine as opposed to part of the on_render() code in AreaRangedAttackHandler so that the entity graphics would not get overwritten by the aoe overlay.